product_docs/docs/pgd/5/cli/configuring_cli.mdx

```diff
@@ -3,33 +3,32 @@ title: "Configuring PGD CLI"
 navTitle: "Configuring PGD CLI"
 ---
 
-PGD CLI can be installed on any system which is able to connect to the PGD cluster. You will require a user with PGD superuser privileges - the [bdr_superuser role](../security) - or equivalent (for example, edb_admin on BigAnimal distributed high-availability) to use PGD CLI.
+PGD CLI can be installed on any system that can connect to the PGD cluster. To use PGD CLI, you need a user with PGD superuser privileges or equivalent. The PGD user with superuser privileges is the [bdr_superuser role](../security). An example of an equivalent user is edb_admin on a BigAnimal distributed high-availability cluster.
 
 ## PGD CLI and database connection strings
 
-You may not need a database connection string. For example, when Trusted Postgres Architect installs the PGD CLI on a system, it also configures the connection to the PGD cluster. This means that PGD CLI will automatically connect when run. 
+You might not need a database connection string. For example, when Trusted Postgres Architect installs the PGD CLI on a system, it also configures the connection to the PGD cluster, which means that the PGD CLI connects when run. 
 
-If you are installing PGD CLI manually, you must give PGD CLI a database connection string so it knows which PGD cluster to connect to. 
+If you're installing PGD CLI manually, you must give PGD CLI a database connection string so it knows which PGD cluster to connect to. 
 
 !!! Important Setting passwords
-PGD CLI does not interactively prompt for your user's password. You must pass your password using one of the following methods:
+PGD CLI doesn't interactively prompt for your password. You must pass your password using one of the following methods:
 
- 1. Adding an entry to your [`.pgpass` password file](https://www.postgresql.org/docs/current/libpq-pgpass.html) which includes the host, port, database name, user name, and password.
- 1. Setting the password in the `PGPASSWORD` environment variable.
- 1. Including the password in the connection string.
+ - Adding an entry to your [`.pgpass` password file](https://www.postgresql.org/docs/current/libpq-pgpass.html), which includes the host, port, database name, user name, and password.
+ - Setting the password in the `PGPASSWORD` environment variable.
+ - Including the password in the connection string.
```

Comment:
Changed to unnumbered list because order isn't relevant. You could consider adding the information about which is recommended directly to that item, like,   
Adding an entry to your [`.pgpass` password file](https://www.postgresql.org/docs/current/libpq-pgpass.html), which includes the host, port, database name, user name, and password. We recommend this option because the other options do't scale well with multiple databases, or they compromise password confidentiality.